RUNNING.md

```diff
@@ -0,0 +1,72 @@
+# Running
+
+## Table of Contents
+
+- [Running With Docker](#running-with-docker)
+- [Environment Variables](#environment-variables)
+- [Troubleshooting](#troubleshooting)
+
+## Running With Docker
+
+For convenience,  `docker-compose.yml` configuration files are supplied with this project.
+The compose file orchestrates the Publisher application and the EQ schema validation service.
+A benefit of this approach is that there is no need to run the schema validation service manually.
+
+### Environment Variables
+
+| Name                   | Description                             | Required |
+| ---------------------- | --------------------------------------- | -------- |
+| `REACT_APP_API_URL`    | Set Author API URL                      | Yes      |
+| `REACT_APP_LAUNCH_URL` | Set the launch-a-survey target          | No       |
+| `PUBLIC_URL`           | The public URL inferred if not provided | No       |
```

Comment:
There are the environment variables for `eq-author`. This isn't clear from the docs.